README.md

```diff
@@ -24,10 +26,10 @@ For example, you can kick off mutation testing using:
 $ stryker run
 ```
 
-Please execute the Styker-CLI commands from your project's root directory.
+Please always execute the Styker-CLI commands from your project's root directory. Although we are able to find your local Stryker installation in parent folders, running Stryker from a different folder than your project's root directory may result in unexpected behavior.
```

Comment:
This is actually becausr of this bug: https://github.com/stryker-mutator/stryker/issues/245